src/endpoint/iam/iam_utils.js

```diff
@@ -81,12 +81,15 @@ function parse_max_items(input_max_items) {
 }
 
 /**
- * validate_params will call the aquivalent function in user or access key
+ * validate_params will call the aquivalent function in user, access key, or tagging
```

Comment:
Good comment, just suggesting to make it an example so we will not need to maintain it.
```suggestion
 * validate_params will call the equivalent function (for example: user, access key, tagging, etc.).
```